examples/experimental/dagster-airlift/examples/tutorial-example/Makefile

```diff
@@ -12,6 +12,14 @@ export TUTORIAL_DBT_PROJECT_DIR := $(MAKEFILE_DIR)/tutorial_example/shared/dbt
 export DBT_PROFILES_DIR := $(MAKEFILE_DIR)/tutorial_example/shared/dbt
 export DAGSTER_URL := http://localhost:3000
 
+# Detect OS and use appropriate date command
```

Comment:
Was this meant to be included in this PR?

Reply:
yea, it's unfortunately necessary to run the `make backfill_today` command, which is useful for debugging